templates/repo/issue/view_content/context_menu.tmpl

```diff
@@ -4,10 +4,15 @@
 		{{svg "octicon-kebab-horizontal"}}
 	</a>
 	<div class="menu">
-		{{ $referenceUrl:= cond (.issue) (Printf "%s%s/issues/%d#%s" AppUrl .ctx.Repository.FullName .ctx.Issue.Index .item.HashTag) (Printf "%s%s/pulls/%d/files#%s" AppUrl .ctx.Repository.FullName .ctx.Issue.Index .item.HashTag) }}
+		{{ $referenceUrl := "" }}
+		{{ if .issue }}
+			{{ $referenceUrl := Printf "%s%s/issues/%d#%s" AppUrl .ctx.Repository.FullName .ctx.Issue.Index .item.HashTag }}
```

Comment:
```suggestion
			{{ $referenceUrl = Printf "%s%s/issues/%d#%s" AppUrl .ctx.Repository.FullName .ctx.Issue.Index .item.HashTag }}
```

Reply:
No ternary but two assignment operators... 😏 Fixed, thank you!